Gems/AtomLyIntegration/EditorModeFeedback/Code/Source/EditorModeFeedbackSystemComponent.cpp

```diff
@@ -75,5 +81,35 @@ namespace AZ
         {
             return m_registeryEnabled;
         }
+
+        void EditorModeFeedbackSystemComponent::OnStartPlayInEditorBegin()
+        {
+            SetEnableRender(false);
+        }
+
+        void EditorModeFeedbackSystemComponent::OnStopPlayInEditor()
+        {
+            SetEnableRender(true);
+        }
+
+        void EditorModeFeedbackSystemComponent::SetEnableRender(bool enableRender)
+        {
+            auto sceneSystem = AzFramework::SceneSystemInterface::Get();
+            if (sceneSystem)
+            {
+                AZStd::shared_ptr<AzFramework::Scene> mainScene = sceneSystem->GetScene(AzFramework::Scene::MainSceneName);
+                if (mainScene)
+                {
+                    AZ::RPI::ScenePtr* rpiScene = mainScene->FindSubsystem<AZ::RPI::ScenePtr>();
+                    if (rpiScene)
+                    {
+                        if (AZ::Render::EditorModeFeatureProcessor* fp = (*rpiScene)->GetFeatureProcessor<AZ::Render::EditorModeFeatureProcessor>())
+                        {
+                            fp->SetEnableRender(enableRender);
+                        }
+                    }
+                }
+            }
+        }
```

Comment:
suggestion: Using early returns with flipped conditions might be nicer here to limit the amount of indentation required